caddyhttp/templates/templates.go

```diff
@@ -62,6 +63,12 @@ func (t Templates) ServeHTTP(w http.ResponseWriter, r *http.Request) (int, error
 					return http.StatusInternalServerError, err
 				}
 
+				ctype := mime.TypeByExtension(ext)
```

Comment:
Hmm, yeah, okay - I've given it some more thought, and I can see why the test isn't effective. We'll need a more advanced test harness to take care of that. Go ahead and remove the test case, but put a comment above this line explaining why we set the Content-Type here. Thanks!